simuleval/options.py

```diff
@@ -161,6 +161,12 @@ def general_parser():
         default="main.yaml",
         help="Name of the config yaml of the system configs.",
     )
+    parser.add_argument(
+        "--tgt-lang",
+        type=str,
+        default=None,
+        help="Path to the Target language file.",
+    )
```

Comment:
I think a more apt place for adding/defining this arg would be in `simuleval/data/dataloader/s2t_dataloader.py`
https://github.com/facebookresearch/SimulEval/blob/50a0783168f98101a427fe89057a7aacaf1b7e1c/simuleval/data/dataloader/dataloader.py#L69

Reply:
The reason it was added in the options.py during the push on github is for the test to pass.

Reply:
The suggestion here is to move the declaration/definition of this `--tgt-lang` arg to `dataloader.py` as this arg is related to the dataloader and makes more contextual sense to have this arg in there instead of `options.py` or `counter_in_tgt_lang_agent.py`.